ServiceProvider/CategoryContentServiceProvider.php

```diff
@@ -14,25 +14,70 @@
 use Eccube\Application;
 use Silex\Application as BaseApplication;
 use Silex\ServiceProviderInterface;
+use Symfony\Component\Translation\Loader\YamlFileLoader;
+use Symfony\Component\Yaml\Yaml;
 
+/**
+ * Class CategoryContentServiceProvider
+ * @package Plugin\CategoryContent\ServiceProvider
+ */
 class CategoryContentServiceProvider implements ServiceProviderInterface
 {
+    /**
+     * register
+     *
+     * @param BaseApplication $app
+     */
     public function register(BaseApplication $app)
     {
         // Form/Extension
-        $app['form.type.extensions'] = $app->share($app->extend('form.type.extensions', function ($extensions) {
-            $extensions[] = new \Plugin\CategoryContent\Form\Extension\CategoryContentExtension();
+        $app['form.type.extensions'] = $app->share($app->extend('form.type.extensions', function ($extensions) use ($app) {
+            $extensions[] = new \Plugin\CategoryContent\Form\Extension\CategoryContentExtension($app['config'], $app);
```

Comment:
Use Plugin\CategoryContent\Form\Extension\CategoryContentExtension;
new CategoryContentExtension();
see in knowhow.